src/main/java/com/trailfinder/dao/EventDAOStub.java

```diff
@@ -15,13 +15,13 @@
 
 	@Override
 	public EventDTO getEvent(int eventId) {
-		// TODO Auto-generated method stub
```

Comment:
TODOs are important to tag work that needs to be done.

You should only delete the TODO when that work is done.  Don't delete it because it's a comment.  

In these methods, the work needed to resolve the TODOs has not occurred, so removing the TODO is not prudent.